app/src/main/java/org/novinger/jason/demoreader/datamodels/Article.java

```diff
@@ -0,0 +1,67 @@
+package org.novinger.jason.demoreader.datamodels;
+
+/**
+ * Created by jason on 4/27/16.
+ */
+public class Article {
+    private String id;
+    private int realId;
+    private String headline;
+    private String slug;
+    private Author[] authors;
+    private String summary;
+    private String url;
+
+    public Article() {}
+
+    public Article(String id, String headline, String slug, Author[] authors, String url, String summary) {
+        this.id = id;
```

Comment:
That was kind of my thought as well. Is the separate package, `org.novinger.jason.demoreader.datamodels` appropriate here to set them apart from the rest of the codebase? I wanted to signal that they were not as ... first-class as the other code.